data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,531 @@
+			`	(Remember to park your other ships and make sure your flagship is the Ancient Ladybug.)`
+				accept
+	on decline
+		"reputation: Hai (Unfettered)" >?= -1000
+	on disabled
+		fail "Hai Legacy 3"
+		dialog `The enemy ships hail you, "You have failed the challenge. We will take the ship from you when we see fit, I heavily suggest you land on Darkcloak so that we take it from you without trouble - we can even leave you with the outfits. Do not try to keep it from us!"`
+	on fail
+		"reputation: Hai (Unfettered)" >?= -1000
+		dialog `The Unfettered hail you, "You failed to complete the duel properly, and as such, are no longer authorized to land on our worlds. You will not enjoy unwarranted respect any longer!"`
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"total ships" == 1
+			has "flagship model: Ancient Ladybug"
+		to despawn
+			"total ships" != 1
+		personality unconstrained launching plunders staying target heroic disables
+		ship "Lightning Bug (Ionic)" "Erazer"
+		ship "Lightning Bug" "Thunder"
+		ship "Lightning Bug (Unfettered Shipyards)" "Lighting"
+		ship "Lightning Bug (Pulse)" "Struck"
+		conversation
+			`You receive a message from the remaining Unfettered ships: "We surrender. If you land now, you will receive your reward, so long as you do not capture our ships."`
+			branch "good boy or bad boy enough"
+				has "unfettered: beaten extra bugs"
+				"old reputation: Hai (Unfettered)" < -2000
+			`	"Do not shout victory too soon, however, for we were not the only ones you had to beat," they add.`
+			label "good boy or bad boy enough"
+			action
+				set "unfettered: beaten the bugs"
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"old reputation: Hai (Unfettered)" < -2000
+			has "flagship model: Ancient Ladybug"
+			"total ships" == 1
+		to despawn
+			"total ships" != 1
+		personality unconstrained entering plunders staying target heroic
+		ship "Sea Scorpion (Shredder)" "Requiem"
+		system "Hi Yahr"
+		dialog "You have managed to defeat the last of the Unfettered challengers, it is now time to land to claim your reward."
+		on disable
+			set "unfettered: beaten extra bugs"
+	npc
+		government "Hai (Unfettered)"
+		ship "Lightning Bug (Unfettered Shipyards)"
+		personality launching staying surveillance
+		on disable
+			dialog `The engineer hails you, "What do you think you are doing? The duel will not be valid if you destroy my ship."`
+		on kill
+			fail "Hai Legacy 3"
+	on visit
+		dialog "You have not beaten all challenging ships yet. If they have not shown up it is probably because you have cheated and brought more ships into the fight. Load an earlier save and try again if you want to finish this mission. You may also have capped too many of their ships, in which case you should disown them."
```

Comment:
Can you explain how the last sentence (about capturing) would occur? How does that affect the mission outcome?

Reply:
"total ships" <= 1 + "unfettered: beaten extra bugs"
so long as you have more than one ship + another if they bent the rule you can't succeed this mission given you didnt respect the no capturing rule
if I could I'd just make this an atrocity but I cant rn until my other PR is in

Reply:
to clarify I playtested it and noticed that it was OP if you captured the lightning bugs so I made sure to not allow that

Reply:
Gotcha, thanks! Just a wording edit then:

```suggestion
		dialog "You have not beaten all challenging ships yet. If they have not shown up it is probably because you have cheated and brought more ships into the fight. Load an earlier save and try again if you want to finish this mission. You may also have captured ships included in the duel, in which case you should disown them."
```

P.S. - Just a heads up, I don't think you use "atrocity" correctly in your reply. Not sure what you intended to say, but I was still able to get the gist.